core/src/main/java/org/apache/sdap/mudrod/weblog/pre/SessionStatistic.java

```diff
@@ -229,7 +229,7 @@ public int processSession(ESDriver es, String sessionId) throws IOException, Int
               String[] keywordList = keywords.split(",");
               for (String item : items) {
                 if (!Arrays.asList(keywordList).contains(item)) {
-                  keywords = keywords + item + ",";
+                  keywords = keywords + "," + item + ",";
```

Comment:
Can you explain this addition? What was wrong with it previously? Do you have a unit test for it?

Reply:
https://github.com/quintinali/incubator-sdap-mudrod/blob/b3ef470a39f8df68ddf80805c6948f870feeb2ce/core/src/test/java/org/apache/sdap/mudrod/discoveryengine/WeblogDiscoveryEngineTest.java#L61 In the test case, I found the result generated from the old code is not the same as expected thus I changed the code

Reply:
Yes